tests/framework/extensions/clusters/clusters.go

```diff
@@ -225,6 +226,9 @@ func NewRKE1ClusterConfig(clusterName string, client *rancher.Client, clustersCo
 				MTU:     0,
 				Options: map[string]string{},
 			},
+			Services: &management.RKEConfigServices{
+				Etcd: clustersConfig.ETCDRKE1,
```

Comment:
This is actually removed as RKE1 isn't supported yet.

Reply:
I don't quite understand. So if it doesn't get set it's removed by rancher? Is that what you mean?

Reply:
I don't think this is the case here. I've done multiple testing with these changes without it causing any overrides.

Reply:
Just to be on the safe side, I removed this and added an if condition similar to what we do for PSACT. If it is not blank in the config file, then use those values. If it is not set, it will function as it did prior to this PR.